tests/Unit/Helpers/GovukPage/ConfirmTest.php

```diff
@@ -0,0 +1,150 @@
+
+use AnthonyEdmonds\GovukLaravel\Helpers\GovukPage;
+use AnthonyEdmonds\GovukLaravel\Pages\Page;
+use AnthonyEdmonds\GovukLaravel\Tests\TestCase;
+
+class ConfirmTest extends TestCase
+{
+    protected array $page;
+
+    protected function makePage(
+        string|null $otherButtonLabel = 'carrot',
+        string|null $otherButtonHref = 'kitten',
+    ): void {
+        $this->page = GovukPage::confirm(
+            'good',
+            'for',
+            'apples',
+            'table',
+            'let',
+            'potato',
+            $otherButtonLabel,
+            $otherButtonHref,
+            'warning',
+            'beagle',
+        )->toArray();
+    }
+
+    public function testHasTitle(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'good',
+            $this->page['title'],
+        );
+    }
+
+    public function testHasBlade(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'for',
+            $this->page['content'],
+        );
+    }
+
+    public function testHasSubmitButtonLabel(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'apples',
+            $this->page['submitButtonLabel'],
+        );
+    }
+
+    public function testHasAction(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'table',
+            $this->page['action'],
+        );
+    }
+
+    public function testHasBack(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'let',
+            $this->page['back'],
+        );
+    }
+
+    public function testHasMethod(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            "potato",
+            $this->page['method'],
+        );
+    }
+
+    public function testHasOtherButtonLabel(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'carrot',
+            $this->page['otherButtonLabel'],
+        );
+    }
+
+    public function testHasOtherButtonHref(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'kitten',
+            $this->page['otherButtonHref'],
+        );
+    }
+
+    public function testHasSubmitButtonType(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'warning',
+            $this->page['submitButtonType'],
+        );
+    }
+
+    public function testHasCaption(): void
+    {
+        $this->makePage();
+
+        $this->assertEquals(
+            'beagle',
+            $this->page['caption'],
+        );
+    }
+
+    public function testOtherButtonHrefIsBackWhenNull(): void
+    {
+        $this->makePage(null, null);
+
+        $this->assertEquals(
+            'let',
+            $this->page['otherButtonHref'],
+        );
+    }
+
+    public function testOtherButtonLabelIsTrueWhenNull(): void
```

Comment:
The test title doesn't quite match up with what is being asserted. This isn't true, it is being checked against a default value. Perhaps something along the lines of `testOtherButtonLabelIsDefaultWhenNull` then reference the default button label const, if there is one.